modules/FIC_FileHandler.py

```diff
@@ -208,3 +208,43 @@ def rename_file_with_date(self, directory, file_name, date_string):
 
         return generated_name
 
+    def is_readable(self, directory_name, file_name):
+        # os.access() is used with two arguments, file_name and os.R_OK to check if the file can be read
+        if os.access(self.construct_path(directory_name, file_name), os.R_OK):
+            self.LOGGER.info("File \"{}\" can be read.".format(self.construct_path(directory_name, file_name)))
```

Comment:
This should be `LOGGER.debug` as this information is useful only if we develop and/or debug the tool.